context/index.js

```diff
@@ -12,12 +12,23 @@ const appSkuConfig = fs.existsSync(appSkuConfigPath)
   : {};
 const skuConfig = merge(defaultSkuConfig, appSkuConfig);
 
+// Validate config
+if (skuConfig.entry.library && !skuConfig.libraryName) {
+  console.log(
+    chalk.red(
+      "Error: In your sku config, you've provided 'entry.library' without a corresponding 'libraryName' option."
```

Comment:
I feel like this could be more to the point and link to the readme for details, eg.
```
"Error: No `libraryName` provided. For more information refer to the `Building a Library` documentation https://github.com/seek-oss/sku#building-a-library"
```

Reply:
Done 👍